client/src/components/ElementName.tsx

```diff
@@ -0,0 +1,48 @@
+import { Text } from "@chakra-ui/react";
+import React, { ReactNode } from "react";
+
+export interface ElementNameProps {
+  element: string;
+  children?: ReactNode
+}
+
+export const ElementName: React.FC<ElementNameProps> = ({
+  element,
+  children
+}) => {
+  const elementColors: Record<string, string> = {
```

Comment:
Is it worth using `ElementType` from mafia here, or does the fact that that includes Sleaze and Shadow and stuff mean it's not worth it?